app/main.py

```diff
@@ -1,8 +1,39 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: str,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: int,
+                 clean_power: int, average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
```

Comment:
You can give more descriptive type annotations

```suggestion
    def serve_cars(self, cars: list[Car]) -> float:
```

Reply:
fix this

Reply:
You still didn't fix this

Reply:
look: type list can have a lot of different types inside. For example, a list of integers, strings, and Car class instances. So we recommend clarifying the list type: `list[int]`, `list[Car]`, ...
In this case, it is the list of Car, so: `list[Car]`